cmd/sync.go

```diff
@@ -7,7 +7,10 @@ import (
 )
 
 var (
-	syncCmdKongStateFile []string
+	// SyncCmdKongStateFile holds kong files passed as an input
+	// NOTE: exported for testing purposes.
+	SyncCmdKongStateFile []string
```

Comment:
If you need to export this, the tests are doing something wrong.
Tests must be able to pass in the appropriate input - filenames in this case - via the main execute command.

Reply:
reworked